README.md

```diff
@@ -22,6 +22,12 @@ config :nadia,
   token: "bot token"
 ```
 
+You can also add an optional recv_timeout in seconds (defaults to 5s).
+```elixir
+config :nadia,
+  opt_recv_timeout: 10
```

Comment:
IMO, recv_timeout is whether default or custom, not optional.
So I'd like to use `recv_timeout` as the config name.